promscale/install-promscale.md

```diff
@@ -0,0 +1,78 @@
+# Install Promscale
+There are several different methods for installing Promscale. This section
+describes installing from a pre-built Docker image. For alternative installation
+methods, see the [Promscale GitHub repository][promscale-github].
+
+If you have a Kubernetes cluster with Helm installed, you can use
+the observability suite for Kubernetes (tobs) to install a full metric
+collection and visualization solution including Prometheus, Grafana, Promscale,
+and a preview version of PromLens. To learn how to do this, watch
+our [demo video][tobs demo video].
+
+If you want to migrate data from Prometheus into Promscale, you can use
+[Prom-migrator][prom-migrator-blog], an open-source, universal Prometheus data
+migration tool that can move data from one remote-storage system to another.
+
+### Procedure: Installing Promscale from a Docker image
+1.  Download the Docker image for the Promscale Connector
```

Comment:
Hmm users don't actually need to download any images, the commands below will do that automatically. This just describes which images are used by the following commands.